update(blog): Staff announcement (#2718)
privacyguides/privacyguides.org: blog/posts/staff-announcement.md

---
date:
    created: 2024-08-20
categories:
    - Announcements
authors:
    - niek-de-wilde
tags:
    - Privacy Guides
license: BY-SA
---

# Jonah Aragon Hired as Project Director

> We are thrilled to announce a significant milestone for Privacy Guides: the addition of our first paid staff member, Jonah Aragon. This achievement is a testament to the unwavering support and generous donations from our incredible community. Another major donation came from [Power Up Privacy](https://powerupprivacy.com/){ rel=nofollow }, a privacy advocacy group that funds privacy-related research and development, which made this goal achievable earlier then expected!<!-- more -->

Reviewer: > which made this goal achievable earlier then expected!

Could we go for 'which helped us achieve this goal earlier than expected' instead? Just reads a little better

---
date:
    created: 2024-08-20
categories:
    - Announcements
authors:
    - niek-de-wilde
tags:
    - Privacy Guides
license: BY-SA
---

> # Privacy Guides Welcomes Its First Staff Member: Jonah Aragon

Reviewer 2: ```suggestion
# Jonah Aragon Hired as Project Director
```

> We are thrilled to announce a significant milestone for Privacy Guides: the addition of our first part-time staff member, Jonah Aragon. This achievement is a testament to the unwavering support and generous donations from our incredible community. Another major donation came from Power Up Privacy, a privacy advocacy group that funds privacy-related research and development, which made this goal achievable earlier then expected!

Reviewer 2: Remember we need the `<!-- more -->` excerpt tag on all blogs:

```suggestion
We are thrilled to announce a significant milestone for Privacy Guides: the addition of our first paid staff member, Jonah Aragon. This achievement is a testament to the unwavering support and generous donations from our incredible community. Another major donation came from [Power Up Privacy](https://powerupprivacy.com/){ rel=nofollow }, a privacy advocacy group that funds privacy-related research and development, which made this goal achievable earlier then expected!<!-- more -->
```

> ## A Community-Driven Success
> 
> The journey of Privacy Guides has always been fueled by the passion and contributions of our community. From the very beginning, our mission has been to provide reliable, independent information on privacy and security. Thanks to your donations, we are now able to take a significant step forward by bringing Jonah on board as our Project Director.
> 
> ## Meet Jonah Aragon
> 
> Jonah Aragon is no stranger to Privacy Guides. As a founding member, Jonah has been instrumental in shaping our organization and its mission. With his extensive background in privacy and cybersecurity, Jonah brings a wealth of knowledge and experience to his new role.

Reviewer 2: ```suggestion
Jonah Aragon is no stranger to Privacy Guides. As a founding member, Jonah has been instrumental in shaping our organization and its mission. With his extensive background in privacy and cybersecurity, Jonah brings a wealth of knowledge and experience to his new role.

## A Community-Driven Success

The journey of Privacy Guides has always been fueled by the passion and contributions of our community. From the very beginning, our mission has been to provide reliable, independent information on privacy and security. Thanks to your donations, we are now able to take a significant step forward by bringing Jonah on board as our Project Director.
```

> ## Jonah’s Role and Responsibilities
> 
> In his new position, Jonah will be working 20 hours a week to manage our day-to-day operations. His responsibilities will include overseeing various projects, ensuring smooth coordination among team members, and maintaining the high standards of our content and resources.
> 
> One of Jonah’s focuses will be restarting our “This Week in Privacy” podcast. This podcast has been a valuable resource for our community, providing weekly updates on the latest developments in the privacy world. After our initial trial run of this project, we heard your positive feedback, so we will be bringing it back!
> 
> ## A Sustainable Future
> 
> Another crucial aspect of Jonah’s role will be fundraising. Our goal is to make this position self-sustaining, ensuring that Privacy Guides can continue to grow and thrive. Jonah’s efforts in fundraising will help secure the necessary resources to support our ongoing projects and initiatives.
> 
> ## Thank You for Your Support

Reviewer 2: ```suggestion
In this new position, Jonah will be working 20 hours a week to manage our day-to-day operations. His responsibilities will include overseeing various projects, ensuring smooth coordination among team members, and maintaining the high standards of our content and resources.

One of Jonah’s focuses will be restarting *This Week in Privacy*, our weekly podcast providing updates on the latest developments in our community and the privacy world. After our initial trial run of this project, we heard your positive feedback, so we will be bringing it back!

Another crucial aspect of Jonah’s role will be fundraising. Our goal is to make this position self-sustaining, ensuring that Privacy Guides can continue to grow and thrive. Jonah’s efforts in fundraising will help secure the necessary resources to support our ongoing projects and initiatives.

## Thank You for Your Support
```
